Test/PerformanceCollector/FunctionalTests/TestApps/TestApp40/Global.asax.cs

```diff
@@ -74,6 +74,7 @@ private static PerformanceCollectorModule InitializePerformanceCollectionModule(
             module.Counters.Add(new PerformanceCounterCollectionRequest(@"Will not parse", "Custom counter - will not parse"));
 
             module.Counters.Add(new PerformanceCounterCollectionRequest(@"\Does\NotExist", "Custom counter - does not exist"));
+            module.Counters.Add(new PerformanceCounterCollectionRequest(@"\Process(??APP_WIN32_PROC??)\IDontExist", "Custom counter with placeholder - does not exist"));
```

Comment:
What about the counter with the name that cannot even be parsed. Do we test those?